### DISCUSSION TITLE
.net framework472制作的webapi服务可以部署到Linux上吗？

### DISCUSSION THREAD

**Discussion by RanZhao233** [BYJRK/DotNet-Discussions#318] [C# 技术交流]
想请教各位大佬有无方法

**Comment by BYJRK** [comment]
想办法迁移到 .NET 6 或 8 吧，Framework 不可能的。可以试试官方的升级助手：https://dotnet.microsoft.com/en-us/platform/upgrade-assistant

**Reply by RanZhao233** [reply]
其实在我提问前就已经知道差不多没戏，不过在网上看到一些说通过docker可以做到，所以心存幻想，升级助手也试过了，依赖项几乎全军覆没，升级甚至不如重开。T_T。  那寒流老师，您觉得用.NET6好点还是.NET8好点，

**Reply by KiyanYang** [reply]
.Net 6 快结束支持了，.Net 8更好

**Reply by RanZhao233** [reply]
> .Net 6 快结束支持了，.Net 8更好

好的，感谢！
